### DISCUSSION TITLE
Show article tags in the preview

### DISCUSSION THREAD

**Discussion by jindradev** [dailydotdev/daily#544] [Feature Requests]
Problem:
Misleading titles, not to mention thumbnails (not to mention GIFs), are sometimes too much and sometimes I just feel like reading my feed and give some eye priority to JavaScript articles. Or I just do not want to search the tag (which I can't even remember). OR like the article and I want to receive more similar articles in my feed. 

Request:
I would like the article tags (which are already present in the triple-dot article menu) to be present in the article card, both in the grid and in the list view. 

Note:
I think this is beneficial to the platform so I would like to see it by default, however, I am sure this is at least "setting option" worthy. 

P.S. Combining some thoughts, I would also add to settings an option to disable thumbnail GIF animation or an option to enable on hover. For this reason, I switched to list view although I like me some nice pictures.

**Comment by idoshamun** [comment]
Hi @jindradev,

Thanks for the thoughtful feedback.
As for the problem you suggest, we're exploring ways to improve our feed by utilizing state-of-the-art recommendation models. It will take some time but I'm positive it should partially address the problem you mentioned.

We're testing a new card design but unfortunately, it doesn't include the tags, as we believe they will add a lot of clutter to the user interface of the feed. We might consider it in the future as an opt-in setting.

A setting for disabling GIFs is a great idea. I'll raise it in our next product planning.
